app/src/main/java/com/hedvig/app/feature/embark/computedvalues/TemplateExpressionCalculator.kt

```diff
@@ -126,7 +126,7 @@ object TemplateExpressionCalculator {
     private fun evaluateAbstractExpression(expression: Expression, store: HashMap<String, String>): String = when (expression) {
         is Expression.StoreKeyExpression -> store[expression.key] ?: ""
         is Expression.NumberConstantExpression -> expression.constant.toBigDecimal().stripTrailingZeros().toString()
-        is Expression.StringConstantExpression -> expression.constant
+        is Expression.StringConstantExpression -> expression.constant.replace("'", "")
```

Comment:
Given that we're matching on a single character, we can use the `char`-based `String::replace` instead of the `String`-based one 🙂 

```suggestion
        is Expression.StringConstantExpression -> expression.constant.replace('\'', '')
```

(with reservations for that this might not compile exactly as suggested 😬)